routes/api.php

```diff
@@ -1,6 +1,7 @@
 <?php
 
 use Illuminate\Http\Request;
+use App\Http\Controllers\VueController;
```

Comment:
This controller is not needed now. So remove it. And then you can merge the PR.

Reply:
ok